packages/story-editor/package.json

```diff
@@ -28,6 +28,7 @@
         "@web-stories-wp/text-sets": "*",
         "@web-stories-wp/tracking": "*",
         "@web-stories-wp/units": "*",
+        "@web-stories-wp/wp-story-editor": "*",
```

Comment:
⚠️  `wp-story-editor` requires `story-editor`, not the other way around.

See my other comment about how not to use `useMediaPicker`